### PR TITLE
Splash Screen Feature

### PR DESCRIPTION
**Pull request by florenciaferraiolo**
## Description
Added a splash screen with a little animation

### Device 1:
video-to-gif-converter (1)

### Device 2:
video-to-gif-converter (2)

## Tested on

Device 1: Nexus S
Device 2: Pixel 1

### CODE REVIEW

File: app/src/main/java/com/example/marvelapp/SplashActivity.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import android.view.View
+import android.view.ViewTreeObserver
+import androidx.appcompat.app.AppCompatActivity
+
+class SplashActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+
+        val content: View = findViewById(android.R.id.content)
```
Comment (mariano98montero) [inline app/src/main/java/com/example/marvelapp/SplashActivity.kt]: please use view binding for this

File: app/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -13,4 +13,4 @@
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
         <!-- Customize your theme here. -->
```
Comment (mariano98montero) [inline app/src/main/res/values/themes.xml]: please remove commented code

**Review by mariano98montero — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/themes.xml (hunk 2 above)
Comment (florenciaferraiolo) [reply app/src/main/res/values/themes.xml]: done

File: app/src/main/java/com/example/marvelapp/SplashActivity.kt (hunk 1 above)
Comment (florenciaferraiolo) [reply app/src/main/java/com/example/marvelapp/SplashActivity.kt]: done

**Review by mariano98montero — APPROVED** [review]

**Pull request merged**